esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -1090,7 +1090,7 @@ def _load_and_preprocess_data(self):
         for dataset in input_data:
             filename = dataset['filename']
             logger.info("Loading %s", filename)
-            cube = iris.load_cube(filename)
+            cube = iris.load_cube(filename, dataset['variable_group'])
```

Comment:
This is very dangerous. `variable_group` is an arbitrary name in the recipe, so this can easily lead to errors. Why is it necessary to change that here? If it's really necessary, the following would be much safer:

```suggestion
            cube = iris.load_cube(filename, dataset['short_name'])
```

Reply:
I had no intention in changing the monitoring diagnostics in any way and have no idea where this came from. I therefore revoked the changes.